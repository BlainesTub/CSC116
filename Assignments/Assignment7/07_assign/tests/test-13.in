(x-6) + (x-10)
5