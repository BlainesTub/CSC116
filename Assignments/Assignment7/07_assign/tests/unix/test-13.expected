f(x) = x - 6 + x - 10
f(5) = -6
f'(x) = 1 - 0 + 1 - 0
f'(5) = 2
